From 1681ce3735483ab6897e403a20a7faf5c34c1f09 Mon Sep 17 00:00:00 2001
From: Johan Hovold <johan+linaro@kernel.org>
Date: Mon, 20 Mar 2023 14:57:09 +0100
Subject: arm64: dts: qcom: sc8280xp-pmics: fix sdam 'reg' property

The SPMI PMIC register region width is fixed and should not be encoded
in the devicetree.

Fixes: d6dbbda37ab5 ("arm64: dts: qcom: sc8280xp-pmics: add pmk8280 sdam nvram")
Signed-off-by: Johan Hovold <johan+linaro@kernel.org>
Reviewed-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Reviewed-by: Konrad Dybcio <konrad.dybcio@linaro.org>
Signed-off-by: Bjorn Andersson <andersson@kernel.org>
Link: https://lore.kernel.org/r/20230320135710.1989-3-johan+linaro@kernel.org
---
 arch/arm64/boot/dts/qcom/sc8280xp-pmics.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sc8280xp-pmics.dtsi b/arch/arm64/boot/dts/qcom/sc8280xp-pmics.dtsi
index 1411edd42b250..c35e7f6bd657a 100644
--- a/arch/arm64/boot/dts/qcom/sc8280xp-pmics.dtsi
+++ b/arch/arm64/boot/dts/qcom/sc8280xp-pmics.dtsi
@@ -107,7 +107,7 @@
 
 		pmk8280_sdam_6: nvram@8500 {
 			compatible = "qcom,spmi-sdam";
-			reg = <0x8500 0x100>;
+			reg = <0x8500>;
 			#address-cells = <1>;
 			#size-cells = <1>;
 			ranges = <0 0x8500 0x100>;
-- 
cgit 

